spec/requests/stories_index_spec.rb

```diff
@@ -1,18 +1,24 @@
 require "rails_helper"
 
 RSpec.describe "StoriesIndex", type: :request do
+  let!(:article) { create(:article, featured: true) }
+
   describe "GET stories index" do
-    it "renders page with proper sidebar" do
+    it "renders page with article list" do
       get "/"
-      expect(response.body).to include("key links")
+      expect(response.body).to include(article.title)
```

Comment:
`CGI.escapeHTML(article.title)` in case the title contains escaped chars, checks against unescaped strings a known source of flakiness in our tests :/

Reply:
Weirdly checking for this broke my tests: Jbuilder appears to not to CGI escaping on values. JSON in general allows most symbols: our current articles feed at https://dev.to/api/articles doesn't do any escaping of titles!